modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc

```diff
@@ -1410,6 +1415,77 @@ neo4j_home$ --nodes persons.csv --nodes games.csv --id-type string
 The `id` property of the nodes in the `persons` group will be stored as `long` type, while the `id` property of the nodes in the `games` group will be stored as `string` type, as the global `id-type` is a string.
 ====
 
+
+== Importing data that spans multiple lines
+
+The `--multiline-fields` option allows fields from an input source to span multiple lines, i.e. contain newline characters.
+For example:
+
+[source, shell, role=noplay]
+----
+bin/neo4j-admin database import full --nodes import/node_header.csv,import/node_data.csv --multiline-fields=true databasename
+----
+
+Where `import/node_data.csv` contains multiline fields, such as:
+
+[source, csv, role=nocopy]
+----
+id,name,birthDate,birthYear, birthLocation, description
+1, John, October 1st, 2000, New York, This is a multiline
```

Comment:
probs best to remove those leading spaces, i.e. `...,October 1st,...` instead of `..., October 1st,...`